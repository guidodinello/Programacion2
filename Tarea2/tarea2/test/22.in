# Implementar ordenar
ordenar
insertarAlFinal (6,2.4)
ordenar
insertarAlFinal (2,3.4)
ordenar
imprimirCadena
ordenar
imprimirCadena
insertarAlFinal (9,3.4)
ordenar
imprimirCadena
finalCadena
insertarSegmentoDespues 3 (5,3.4) (11,9.4) (7,2.6)
imprimirCadena
ordenar
imprimirCadena
Fin
