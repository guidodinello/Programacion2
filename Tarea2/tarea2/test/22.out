1>#  Implementar ordenar.
2>Quedó ordenada.
3>Insertado al final.
4>Quedó ordenada.
5>Insertado al final.
6>Quedó ordenada.
7>(2,3.40)(6,2.40)
8>Quedó ordenada.
9>(2,3.40)(6,2.40)
10>Insertado al final.
11>Quedó ordenada.
12>(2,3.40)(6,2.40)(9,3.40)
13>loc al final.
14>Segmento insertado después de loc.
15>(2,3.40)(6,2.40)(9,3.40)(5,3.40)(11,9.40)(7,2.60)
16>Quedó ordenada.
17>(2,3.40)(5,3.40)(6,2.40)(7,2.60)(9,3.40)(11,9.40)
18>Fin.
